tests/smoke/rest.smoke.test.ts

```diff
@@ -656,24 +656,28 @@ describe('HiFi API REST Calls', () => {
             let attenuation4Data: AttenuationData;
             attenuation1Data = {
                 "attenuation": 0.5,
+                "frequency-rolloff": 12,
                 "listener-zone-id": zone1Data.id,
                 "source-zone-id": zone2Data.id,
                 "za-offset": -5
             };
             attenuation2Data = {
                 "attenuation": 0.5,
+                "frequency-rolloff": 12,
                 "listener-zone-id": zone1Data.id,
                 "source-zone-id": zone2Data.id,
                 "za-offset": -5
             };
             attenuation3Data = {
                 "attenuation": 0.5,
+                "frequency-rolloff": 12,
                 "listener-zone-id": zone1Data.id,
                 "source-zone-id": zone3Data.id,
                 "za-offset": -5
             };
             attenuation4Data = {
                 "attenuation": 0.5,
+                "frequency-rolloff": 12,
                 "listener-zone-id": zone1Data.id,
                 "source-zone-id": zone4Data.id,
                 "za-offset": -5
```

Comment:
We should try testing `null` as a value for one of the frequency rolloffs and one of the attenuations, now that that's a feature.

Reply:
Actually, when I tried inputting null, the fetch request failed.